src/kw_db.sh

```diff
@@ -0,0 +1,30 @@
+# This file handles the interactions with the kw database
+# It should be used to read, write and modify data in kw's dbs
+
+include "$KW_LIB_DIR/kwio.sh"
+include "$KW_LIB_DIR/kwlib.sh"
+
+declare -g DB_NAME='kw.db'
+
+# This function reads and executes a SQL script in the database
+#
+# @sql_path:  Path to SQL script
+# @db:        Name of the database file
+# @db_folder: Path to the folder that contains @db
+#
+# Return:
+# 0 if succesful; non-zero otherwise
+function execute_sql_script()
+{
+  local sql_path="$1"
+  local db="${2:-"$DB_NAME"}"
+  local db_folder="${3:-"$KW_DATA_DIR"}"
+  local db_path
+
+  db_path="$(join_path "$db_folder" "$db")"
+
+  file_exists "$sql_path" || exit_msg "Could not find the file: $sql_path" 2
+  file_exists "$db_path" || warning "Creating database: $db_path"
```

Comment:
I'm thinking of removing the message from that function and only check if the file exists and return accordingly. And now I think it might be easier to just use `[[ -f "$path" ]]`. What do you think?

Reply:
I agree